src/quantum/azext_quantum/operations/offerings.py

```diff
@@ -0,0 +1,74 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long,redefined-builtin
+
+from knack.util import CLIError
+from .._client_factory import cf_offerings, cf_vm_image_term
+import time
+
+PUBLISHER_NOT_AVAILABLE = "N/A"
+OFFER_NOT_AVAILABLE = "N/A"
+
+
+def _show_info(msg):
+    import colorama
+    colorama.init()
+    print(f"\033[1m{colorama.Fore.GREEN}{msg}{colorama.Style.RESET_ALL}")
+
+
+def _get_publisher_and_offer_from_provider_id(providers, provider_id):
+    publisher_id = None
+    offer_id = None
+    for p in providers:
+        if (p.id.lower() == provider_id.lower()):
+            offer_id = p.properties.managed_application.offer_id
+            publisher_id = p.properties.managed_application.publisher_id
+            break
+    return (publisher_id, offer_id)
+
+
+def _valid_publisher_and_offer(provider, publisher, offer):
+    if (offer is None or publisher is None):
+        raise CLIError(f"Provider '{provider}' not found.")
+    if (offer == OFFER_NOT_AVAILABLE or publisher == PUBLISHER_NOT_AVAILABLE):
+        _show_info(f"No terms exist for provider '{provider}'.")
```

Comment:
Is it an error? We seldom use raw `print`.

Reply:
In this case it's not an error. We will end up returning None from the command, but the message is just a way to let the user know that this is not an error, simply there are no terms to accept for this provider.

Reply:
I'll add a clarifying comment and improve the message of the print output.

Reply:
OK. It is like logger.info or logger.warning. But you want to customize color. No problem.